app/views/layouts/admin_v2/base.html.erb

```diff
@@ -11,6 +11,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta property="og:image" content="<%= image_url('og-logo.png') %>">
+    <meta name="turbo-refresh-method" content="morph">
+    <meta name="turbo-refresh-scroll" content="preserve">
```

Comment:
This was added to preserve the scroll position when using the change password form at the bottom of the settings page.